release_notes/topics/9_0_0.adoc

```diff
@@ -14,6 +14,16 @@ A new built-in vault provider that reads secrets from a keystore-backed Elytron
 extension. The creation and management of the credential store is handled by Elytron using either the `elytron` subsystem or the
 `elytron-tool.sh` script.
 
+== More updates to W3C WebAuthn and Authentication flows
+
+In this release, we did some usability improvements to the authentication flows. It should be more easy for end user to choose between
```

Comment:
change 'more easy for end user"  to "easier for the end user"

Reply:
Fixed